src/glang/lib/std/io.c3

```diff
@@ -53,13 +74,13 @@ function flush : (io : BufferedFile mut&) -> void = {
     let data_to_write : View<u8> = {&io.buffer, io.buffer_pointer};
     let string_to_write : StringView = {data_to_write};
     // UFCS where `write` takes a `T&` and we have a `T mut&`
-    (&io.underlying_file):write(data_to_write);
+    (&io.underlying):write(data_to_write);
```

Comment:
Can we do `io.underlying:write(data_to_write` now? (here and below)

Reply:
ooo, I forgot about this :heart_eyes: